spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -936,6 +936,11 @@ public String getSparkUIUrl() {
       return sparkUrl;
     }
 
+    String sparkUrlProp = property.getProperty("zeppelin.spark.uiWebUrl", "");
+    if (!sparkUrlProp.isEmpty()) {
```

Comment:
let's check for isBlank https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#isBlank-java.lang.CharSequence-

Reply:
I agree. Fixed on latest commit